geps/gep-3793/index.md

```diff
@@ -189,10 +194,274 @@ will need resolution before this GEP can graduate.
+`parentRefs` MUST fail validation. If a Route with an empty array for
+`parentRefs` somehow exists in the cluster, all Gateways in the cluster MUST
+refuse to accept it. (Omitting `parentRefs` entirely will work much more
+cleanly with GitOps tools than specifying an empty array.)
+
+Note also that if Ana specifies _any_ `parentRefs`, the default Gateway MUST
+NOT claim the Route unless of the `parentRefs` explicitly names the default
+Gateway. To do otherwise makes it impossible for Ana to define mesh-only
+Routes, or to specify a Route that is meant to use only a specific Gateway
+that is not the default. This implies that for Ana to specify a Route intended
+to serve both north/south and east/west roles, she MUST explicitly specify the
+Gateway in `parentRefs`, even if that Gateway happens to be the default
+Gateway.
+
+All other characteristics of a Route using the default Gateway MUST behave the
+same as if the default Gateway were explicitly specified in `parentRefs`.
+
+The default Gateway MUST use `status.parents` to announce that it has bound
+the Route, for example:
+
+```yaml
+status:
+  parents:
+  - name: my-default-gateway
+    namespace: default
+    controllerName: gateway.networking.k8s.io/some-gateway-controller
+    conditions:
+    - type: Accepted
+      status: "True"
+      lastTransitionTime: "2025-10-01T12:00:00Z"
+      message: "Route is bound to default Gateway"
+```
+
+The default Gateway MUST NOT rewrite the `parentRefs` of a Route using the
+default Gateway; it MUST leave `parentRefs` empty. This becomes important if
+the default Gateway changes, or (in some situations) if GitOps tools are in
+play.
+
+##### Enumerating Routes Bound to the Default Gateway
+
+To enumerate Routes bound to the default Gateway, Ana can look for Routes with
+no `parentRefs` specified, and then check the `status.parents` of those Routes
+to see if the Route has been claimed. This will also tell Ana which Gateway is
+the default, even if she doesn't have RBAC to query Gateway resources
+directly.
+
+While this is possible with `kubectl get -o yaml`, it's not exactly a friendly
+user experience, so adding this functionality to a tool like `gwctl` would be
+a dramatic improvement. In fact, looking at the `status` of a Route is very
+much something that we should expect Ana to do often, whether or not default
+Gateways are in play; `gwctl` or something similar SHOULD be able to show her
+which Routes are bound to which Gateways in every case, not just with default
+Gateways.
+
+##### Open Questions
+
+Should the Gateway also add a `condition` explicitly expressing that the Route
+has been claimed by the default Gateway, perhaps with `type: DefaultGateway`?
+This could help tooling like `gwctl` more easily enumerate Routes bound to the
+default Gateway.
+
+#### 2. Controlling which Gateway is the Default
+
+Since Chihiro must be able to control which Gateway is the default, selecting
```

Comment:
The language here suggests that there will be a "default Gateway", but realistically do we have to support the notion of "default Gateway**s**"?